bin/create-k8s-config-maps-and-secrets

```diff
@@ -31,6 +31,19 @@ data:
   CMR_ENDPOINT: "${CMR_ENDPOINT}"
   USER_AGENT: "${USER_AGENT}"
   SERVICES_YML: $(echo -n "${SERVICES_YML}" | base64 | tr -d "\n")
+  ASF_GDAL_SUBSETTER_IMAGE: "${ASF_GDAL_SUBSETTER_IMAGE}"
```

Comment:
This makes me uneasy, since it's yet another thing to remember to do if we add a new service.  Do we have any docs started on what it takes to add a service that need to be updated?  Is there any way to automate this? (Hack: dump env vars ending with `_IMAGE`)

Reply:
Good idea. Done.